src/WorldMap.cpp

```diff
@@ -0,0 +1,42 @@
+#include "WorldMap.h"
+#include <iostream>
+#include <algorithm>
+#include <fstream>
+#include <sstream>
+#include <assert.h>
+#include <functional>
+
+WorldMap::WorldMap()
+	: m_tileLayers(),
+	m_currentMapName(),
+	m_tileSheetDetails(32, 40, 54, 0, 0)
```

Comment:
If we're decoding the rest of the tile sheet data from a file, we should decode these properties from the file as well.